composer.json

```diff
@@ -21,7 +21,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "dev-master",
-		"phpunit/php-file-iterator": "1.3.3"
+		"phpunit/php-file-iterator": "1.3.3",
+		"phpunit/phpunit": "~4.0"
```

Comment:
The Hamcrest doesn't use any PHPUnit version-specific stuff, so it can run on PHPUnit 3.0 as well. This way forcing library contributors to have 2 phpunit versions installed (the global one and one that comes with a library) only creates a confusion at the end.

I recommend you remove it and please use latest PHPUnit version.